src/main/java/pl/nogacz/snake/board/Board.java

```diff
@@ -28,22 +29,72 @@
     private int tailLength = 0;
 
     private Coordinates snakeHeadCoordinates = new Coordinates(10, 10);
+    private Coordinates rottenAppleCurrentCoordinates;
+    private Coordinates rottenAppleCurrentCoordinates2;
+    private Coordinates rottenAppleCurrentCoordinates3;
+
+    long lastSpawnTime;

```

Comment:
Aynı yorumum buradaki tüm 2, 3 değişkenler için geçerli.

Reply:
done.